internal/httphandler/provenance_request_handler_test.go

```diff
@@ -635,7 +635,7 @@ func TestGetTxIDsSubmittedBy(t *testing.T) {
 			dbMockFactory: func(response interface{}) bcdb.DB {
 				db := &mocks.DB{}
 				db.On("GetCertificate", submittingUserName).Return(aliceCert, nil)
-				db.On("GetTxIDsSubmittedByUser", targetUserID).Return(nil, errors.New("error in provenance db"))
+				db.On("GetTxIDsSubmittedByUser", submittingUserName, targetUserID).Return(nil, errors.New("error in provenance db"))
```

Comment:
add a test case that returns permission error and check that the handler returns 403, in all variants of `Get*ByUser`

Reply:
Done